New Docs Site deployment structure info (#20)
RockefellerArchiveCenter/docs-guide: using-github.md

3. Format the name using lowercase letters and dashes between words "short-name" (e.g. processing-manual).
4. Add a short description of the documentation.
5. Choose to make the repo public or private based on its pre-approved designation (see the [Rockefeller Archive Center Documentation Site Content Approval Policy](http://docs.rockarch.org/docs-policy/)). The designation must be approved by the RAC Assistant Director for the relevant function area, and documentation that is made publicly available must also be approved by the Director of Archives.

> 6. Add a [GitHub Webhook](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks) to the repository. Webhooks are added in the Settings page of a repository. The Webhook listens for `push` events which push the updates to the development and production servers, and it should point to the AWS API Gateway endpoint - build_docs_site_api. Contact the Head of Digital Strategies for the GH_TOKEN as part of the webhook configuration. For more information, see the [Documentation Site Info Sheet](https://docs.rockarch.org/systems-info-sheets/documentation-site-info-sheet/).

Reviewer: ```suggestion
6. Add a [GitHub Webhook](https://docs.github.com/en/developers/webhooks-and-events/webhooks/about-webhooks) to the repository. Webhooks are added in the Settings page of a repository. The Webhook listens for `push` events which push the updates to the development and production servers, and it should point to the AWS API Gateway endpoint for the `build_docs_site_api`. Contact the Head of Digital Strategies for the token and AWS Gateway endpoint URL you will need as part of the webhook configuration. For more information, see the [Documentation Site Info Sheet](https://docs.rockarch.org/systems-info-sheets/documentation-site-info-sheet/).
```
